marquez_client/client.py

```diff
@@ -338,8 +328,15 @@ def _post(self, url, payload, as_json=True):
 
         response = requests.post(
             url=url, headers=_HEADERS, json=payload, timeout=self._timeout)
-        log.info(f'{url}', method='POST', payload=json.dumps(
-            payload), duration_ms=(self._now_ms() - now_ms))
+
+        post_details = {}
```

Comment:
I don't think the `post_details` is still needed? Think that was part of your initial exploration for logging req/res to a file.

Reply:
only when logging is enabled.